src/FourBitTwoDisclosureDeviceUnlocker.java

```diff
@@ -179,10 +176,10 @@ private static boolean isValidSpin(final int numOfSpins) {
         boolean continueSpin;
```

Comment:
In implementing code review changes the if(dev == null) statement got taken out. If this is intentional, please ignore the rest. Otherwise, slip the null check  into isValidSpin().

Reply:
Good eye. I meant to do this and completely forgot.